lib/atoms/a/a.component.js

```diff
@@ -0,0 +1,13 @@
+var lnAA = {
+    templateUrl: 'lnPatterns/atoms/a/template.html',
+    bindings: {
+        link: '@',
+        href: '@',
+        target: '@',
+        class: '@'
+    }
+};
+
+angular
```

Comment:
Can you put this above the var declaration to be consistent with our other components?